build.mjs

```diff
@@ -100,15 +95,17 @@ try {
                                 },
                             },
                             env: {
-                                targets: "defaults",
+                                targets: "fully supports es6",
                                 include: [
-                                    "transform-classes",
-                                    "transform-arrow-functions",
                                     "transform-block-scoping",
-                                    "transform-class-properties"
+                                    "transform-classes"
                                 ],
                                 exclude: [
```

Comment:
Exclude `transform-async-to-generator` since we already have `transform-arrow-functions` (Why should we even exclude this? Shouldn't ES6 support `async` syntax?